src/shared/cache-types.cts

```diff
@@ -154,3 +158,60 @@ export type CacheHandlerForMultipleVersions = BaseCacheHandlerForMultipleVersion
     context: CacheHandlerSetContextForMultipleVersions,
   ) => ReturnType<BaseCacheHandlerForMultipleVersions['set']>
 }
+
+export type TagManifest = { revalidatedAt: number }
+
+export type HtmlBlob = {
+  html: string
+  isFallback: boolean
+}
+
+export type BlobType = NetlifyCacheHandlerValue | TagManifest | HtmlBlob
+
+const isTagManifest = (value: BlobType): value is TagManifest => {
+  return false
+}
+
+const isHtmlBlob = (value: BlobType): value is HtmlBlob => {
+  return false
+}
+
+export const estimateBlobSize = (valueToStore: BlobType | null | Promise<unknown>): number => {
+  // very approximate size calculation to avoid expensive exact size calculation
+  // inspired by https://github.com/vercel/next.js/blob/ed10f7ed0246fcc763194197eb9beebcbd063162/packages/next/src/server/lib/incremental-cache/file-system-cache.ts#L60-L79
+  if (valueToStore === null || isPromise(valueToStore) || isTagManifest(valueToStore)) {
+    return 25
+  }
+  if (isHtmlBlob(valueToStore)) {
```

Comment:
I might be reading the function incorrectly, but it seems like `isHtmlBlob` always returns false

Reply:
Ops, yes - this is actually missing - thanks for spotting. This would result in getting into fallback and produce warnings (tho things would work)

Reply:
Hopeffully fixed in https://github.com/opennextjs/opennextjs-netlify/pull/2777/commits/578fbc54fb33f29febea2bf770af713e817c19b9 with some added unit tests